cypress/support/helpers/runTestsForPage.js

```diff
@@ -89,6 +89,10 @@ const runTestsForPage = ({
         describe(`${pageType} - ${currentPath} - AMP`, () => {
           before(() => {
             Cypress.env('currentPath', currentPath);
+            if (pageType === 'frontPage') {
+              // eslint-disable-next-line no-param-reassign
+              currentPath += '?site=test';
```

Comment:
Is it worth commenting here to explain why this was done. I think it may look a bit of a mystery parameter later on without the context?

Reply:
Yes good idea, I'll add a comment

Reply:
Done